File: Keepr/Models/VaultKeep.cs
namespace Keepr.Models;

    public class VaultKeep : RepoItem<int>
    {
        public Vault Vault { get; set; }
        public int VaultKeepId { get; set; }
        public string CreatorId { get; set; }
        public Profile Creator { get; set; }
        public int VaultId { get; set; }
        public int KeepId { get; set; }
    }

File: Keepr.client/src/models/VaultKeep.js
import { Keep } from "./Keep"


export class VaultKeep {
    constructor(data) {
        if (data.keep) {
            this.keep = data.keep
        }
        this.name = data.name
        this.description = data.description
        this.id = data.id
        this.img = data.img
        this.vaultKeepId = data.vaultKeepId
        this.vaultId = data.vaultId
        this.keepId = data.keepId
        this.creatorId = data.creatorId
    }
}